appium/api-mocks/apis/google/google-messages.ts

```diff
@@ -1,4 +1,4 @@
 /* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
 
-export type GoogleMockAccountEmail = 'e2e.enterprise.test@flowcrypt.com' | 'e2e.consumer.tests@outlook.com';
-export type GoogleMockMessage = 'CC and BCC test' | 'Your FlowCrypt Backup';
+export type GoogleMockAccountEmail = 'e2e.enterprise.test@flowcrypt.com' | 'flowcrypt.compatibility@gmail.com';
+export type GoogleMockMessage = 'CC and BCC test' | 'Test 1' | 'Signed and encrypted message' | 'Honor reply-to address - plain' | 'email with text attachment' | 'Message with cc and multiple recipients and text attachment' | 'new message for reply' | 'Signed only message' | 'Signed only message with detached signature' | 'Signed only message where the pubkey is not available' | 'Signed only message that was tempered during transit' | 'Partially signed only message' | 'encrypted - MDC hash mismatch - modification detected - should fail' | 'message encrypted for another public key (only one pubkey used)' | 'wrong checksum' | 'not integrity protected - should show a warning and not decrypt automatically' | 'key mismatch unexpectedly produces a modal' | 'test thread rendering' | 'Archived thread';
```

Comment:
Let's add some newlines :-) for each line to have reasonable length.

Reply:
Agree, looks bad for me too :)